lua/lvim/config/defaults.lua

```diff
@@ -1,5 +1,6 @@
 return {
   leader = "space",
+  reload_config_on_save = false,
```

Comment:
```suggestion
  reload_config_on_save = true,
```
I don't see the need to disable this by default, as that will make this PR a breaking change.
+
we worked so hard to have this standout feature be stable, this will get it buried :smiling_face_with_tear: